modules/arm_plugin/thirdparty/CMakeLists.txt

```diff
@@ -126,7 +129,7 @@ else()
 
     list(APPEND ARM_COMPUTE_OPTIONS extra_cxx_flags=${extra_cxx_flags})
 
-    if (ARM_COMPUTE_WERROR)
+    if (NOT ARM_COMPUTE_WERROR STREQUAL "")
```

Comment:
why do we need to compare with `""`? I suppose everything is OK if we make this option to be `BOOL`